src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -7745,12 +7745,13 @@ def test_update_gallery_permissions(self, resource_group):
             self.check('sharingProfile.permissions', 'Private')
         ])
 
-        # update gallery from private to community
-        self.cmd('sig update -g {rg} --gallery-name {gallery1} --permissions Community '
-                 '--publisher-uri pubUri --publisher-email test@123.com --eula eula --public-name-prefix pubName',
-                 checks=[
-                     self.check('sharingProfile.permissions', 'Community')
-                 ])
+        # Update gallery from private to community.
+        # This command is commented out because the backend service currently prevents updating 'Private' to 'Community'.
+        # self.cmd('sig update -g {rg} --gallery-name {gallery1} --permissions Community '
+        #          '--publisher-uri pubUri --publisher-email test@123.com --eula eula --public-name-prefix pubName',
+        #          checks=[
+        #              self.check('sharingProfile.permissions', 'Community')
+        #          ])
```

Comment:
Why do we need to remove this test?

Reply:
I have added a comment for it:
![image](https://github.com/user-attachments/assets/c015501a-0480-49fc-b788-56b9f9f9745c)
It is the same as https://github.com/Azure/azure-cli/blob/3b9a7c29581654d9b8d7c79271eaf4105bb92852/src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py#L6625-L6627